portal-web/docroot/html/common/themes/bottom_js.jspf

```diff
@@ -72,8 +72,10 @@
 	}
 </aui:script>
 
-<aui:script require="metal-dom/src/all/dom as dom" sandbox="<%= true %>">
-	var focusInPortletHandler = dom.delegate(
+<aui:script require="frontend-js-web/liferay/delegate/delegate.es as delegateModule" sandbox="<%= true %>">
+	var delegate = delegateModule.default;
+	
+	var focusInPortletHandler = delegate(
```

Comment:
It seems nobody is doing anything with handler here (same for `focusOutPortletHandler`), which means you should probably get rid of the `var` bindings and just leave the calls to `delegate()`.